src/contracts/Protocol/QuestionAPI.sol

```diff
@@ -130,6 +163,8 @@ contract QuestionAPI is Ownable, NFTLocked {
 
         // Publish the question
         _questionStateController.publish(questionId);
+
+        emit QuestionPublished(questionId);
```

Comment:
should we throw the _msgSender() in here too?  for consistency and to track which admin hit publish?